mmv1/products/compute/UrlMap.yaml

```diff
@@ -304,6 +314,66 @@ properties:
           description: |
             An optional description of this resource. Provide this property when you create
             the resource.
+        - !ruby/object:Api::Type::NestedObject
+          name: 'defaultCustomErrorResponsePolicy'
```

Comment:
Should the subvalues here be required, or can they be omitted? If they can be omitted, can we do that in some of the tests?

Reply:
I believe it's an optional value yes. I will make it optional and include it in a test